q2_vsearch/plugin_setup.py

```diff
@@ -249,6 +251,9 @@
                          'longer sequences, it is clustered with the shortest '
                          'of them. If they are equally long, it is clustered '
                          'with the most abundant.'),
+        'min_seq_length': ('Discard sequences shorer than this integer.'),
```

Comment:
Probably supposed to be shorter not shorer? Other than that this seems good. Just exposing more vsearch functionality right?

Reply:
Fixed!

Yes, this should be a simple PR, most of the commits are me messing with hashing or trying to get testing and linting to work.